Makefile

```diff
@@ -38,7 +38,7 @@ ACN_VERSION				?= $(shell git describe --exclude "azure-ipam*" --exclude "dropgz
 AZURE_IPAM_VERSION		?= $(notdir $(shell git describe --match "azure-ipam*" --tags --always))
 CNI_VERSION				?= $(ACN_VERSION)
 CNI_DROPGZ_VERSION		?= $(notdir $(shell git describe --match "dropgz*" --tags --always))
-CNS_VERSION				?= $(ACN_VERSION)
+CNS_VERSION				?= $(ACN_VERSION)-p
```

Comment:
Is this intentional ?

Reply:
yeah added this for my local CNS image to be different than the old one. Removed it from the PR